security/engine/README.md

```diff
@@ -0,0 +1,13 @@
+# CEL Engine
```

Comment:
Can you make all the files in "security/rbac/engine/..." please? we might have some other code modules. The README, go.mod, and go.sum can be in "security/rbac", and all the code related to engine shall be in "security/rbac/engine"

Reply:
would the module be named rbac then? with a submodule named engine?

Reply:
security/rbac will be a submodule, and engine will be a package within the submodule.